mailbox/elasticsearch/src/test/java/org/apache/james/mailbox/elasticsearch/ElasticSearchIntegrationTest.java

```diff
@@ -104,4 +122,73 @@ protected void initializeMailboxManager() throws Exception {
         storeMailboxManager.setMessageSearchIndex(messageSearchIndex);
         storeMailboxManager.init();
     }
+
```

Comment:
I would add a test 'reasonableLongTermShouldNotBeIgnored' where I would index a mail with the following term "dichlorodiphényltrichloroéthane"